doc/setup.md

```diff
@@ -19,6 +19,8 @@ symfony new . 2.8
 composer require victoire/victoire \
     friendsofsymfony/user-bundle:~2.0@dev \
     doctrine/orm \
+    doctrine/doctrine-fixtures-bundle \
+    symfony/assetic-bundle \
```

Comment:
If you use Symfony installer as said in the step before, assetic will be installed automatically